CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log for rocm-cmake
 
+## [0.6.2]
+### Changed
+- If the ROCm platform version that is being built for is less than version 4.5.0, `ROCM_DEP_ROCMCORE` is automatically disabled.
+  - The ROCm platform version is determined by:
+    - the user-set CMake variable `ROCM_PLATFORM_VERSION`; otherwise
```

Comment:
Just looking at the code, I only see commands to set ROCM_PLATFORM_VERSION, but without a test to see if it is already set. Am I missing something?

Reply:
As `ROCM_PLATFORM_VERSION` is a cache variable, it is only set if there isn't an existing cache entry, so if the user has already set the value then the set commands do nothing.